core/src/smartcontracts/wasm.rs

```diff
@@ -172,11 +172,55 @@ impl<'wrld, W: WorldTrait> State<'wrld, W> {
     }
 }
 
+struct IrohaModule {
+    module: Module,
+}
```

Comment:
This struct represents the intermediary module with the host to manage memory allocation requests from a smartcontract module, right?

Reply:
yes, that's it